src/main/fabric8/route.yml

```diff
@@ -0,0 +1,15 @@
+metadata:
+  annotations:
+    app.kubernetes.io/vcs-uri: git@github.com:snowdrop/rest-http-example.git
+  labels:
+    app.kubernetes.io/part-of: http-api-example
+    app.kubernetes.io/runtime-version: ${spring-boot.version}
+    app.kubernetes.io/runtime: spring
```

Comment:
Should the runtime be `Spring` or `Spring Boot` instead ? 
Can you check please with Naina which value we should use ?
Is the label runtime linked to the spring or Spring boot icon to graphically display it when it is selected within the OpenShift DevConsole ?

Reply:
@nainaz what value should we use?

Reply:
In Rodney's table the suggested values are: nodejs | spring | thorntail | vertx

Reply:
FYI: The launcher catalog of the examples uses spring-boot -> https://github.com/fabric8-launcher/launcher-booster-catalog/blob/master/catalog.json#L889

Reply:
In that case maybe we should stick to it.

Reply:
Actually that's not correct. We should use `spring` because that's the value used to add an icon by Che

Reply:
It's quite misleading. In here they say spring https://docs.jboss.org/display/OBST/Labels+needed+for+Examples and it is used in che here https://github.com/christianvogt/console/blob/master/frontend/public/components/catalog/catalog-item-icon.tsx#L174. However, it here it's spring-boot https://github.com/gorkem/app-labels/blob/master/labels-annotation-for-openshift.adoc